adoc/Private-Registry-Installation.adoc

```diff
@@ -91,11 +97,11 @@ By default, {spr} has all supported components enabled. Some components may be d
 +
 --
 [loweralpha]
-. `trivy` - may be disabled, if security vulnerability scanning features are not required
-. `notary` - may be disabled, if artifact signing features are not required
+. `trivy` - disable if you do not require security vulnerability scanning features.
+. `notary` - disable if you do not require artifact-signing features.
```

Comment:
Isn't it actually _feature_ (singular) in both cases? Scanning feature + signing feature....

Reply:
Good point. It means an extra article, but it is correct. Changed.